uhubctl.c

```diff
@@ -1053,7 +1060,10 @@ int main(int argc, char *argv[])
                         int port_status = get_port_status(devh, port);
                         int power_mask = hubs[i].super_speed ? USB_SS_PORT_STAT_POWER
                                                              : USB_PORT_STAT_POWER;
-                        if (k == 0 && !(port_status & power_mask))
+                        if (opt_action == POWER_TOGGLE) {
+                            request = (port_status & power_mask) ? LIBUSB_REQUEST_SET_FEATURE : LIBUSB_REQUEST_CLEAR_FEATURE;
```

Comment:
I tested this locally, this condition is opposite to what it should be. Please change to following (works for me):
```
request = (port_status & power_mask) ? LIBUSB_REQUEST_CLEAR_FEATURE
                                     : LIBUSB_REQUEST_SET_FEATURE;
```

Reply:
Thank you 👍! Resolved …